app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -82,10 +75,17 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val SMALL_TEXT_SIZE = 0
-private const val MEDIUM_TEXT_SIZE = 5
-private const val LARGE_TEXT_SIZE = 10
+private const val SMALL_TEXT_SIZE_SCALE = 0.8f
+private const val MEDIUM_TEXT_SIZE_SCALE = 1.0f
+private const val LARGE_TEXT_SIZE_SCALE = 1.2f
+private const val EXTRA_LARGE_TEXT_SIZE_SCALE = 1.4f
 
+private const val SMALL_TEXT_SIZE_INDEX = 0
+private const val MEDIUM_TEXT_SIZE_INDEX = 1
+private const val LARGE_TEXT_SIZE_INDEX = 2
+private const val EXTRA_LARGE_TEXT_SIZE_INDEX = 3
+
+/** Tests for [ReadingTextSizeFragment]. */
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(application = ReadingTextSizeFragmentTest.TestApplication::class)
```

Comment:
Add qualifiers

Reply:
Do we still need to add qualifiers?

Reply:
We keep in all test files, for stabilizing robolectric.